Add localisation fixing file

TheScienceDemon opened the pull request:
## What
This file (which was requested by trainvoi) sets, upon execution, all quests' name and desc localisation keys to the quest's ID.

ex:
Quest No. **123** - name / desc: "susy.quest.db.**135**.title/desc"
Upon exection:
Quest No. **123** - name / desc: "susy.quest.db.**123**.title/desc"

Useful when having deleted a quest. (As was the case with trainvoi)

## Outcome
Added a file (`questbookLocalisationShift.py`) under `Supersymmetry\build`.

TheScienceDemon committed:
Add fix file

This file (which was requested by trainvoi) sets, upon execution, all quests' name and desc localisation keys to the quest's ID.

ex:
Quest No. 123 - name / desc: "susy.quest.db.135.title/desc"
Upon exection:
Quest No. 123 - name / desc: "susy.quest.db.123.title/desc"

Useful when having deleted a quest. (As was the case with trainvoi)

TheScienceDemon committed:
Change an `and` to `or`

Now either the title or the desc need to be wrong, not both at the same time

bruberu commented on build/questbookLocalisationShift.py:
```diff
@@ -0,0 +1,46 @@
+		questId = questbook[location][quest]["questID:3"]
+		print(f"Found quest with ID: {questId}")
+
+		wantedQuestTitel = "susy.quest.db.%s.%s" % (questId, "title")
```
This variable name does appear to have a spelling error in it? It's also copied throughout the rest of the code.

bruberu commented on build/questbookLocalisationShift.py:
```diff
@@ -0,0 +1,46 @@
+		actualQuestDescription = questbook[location][quest]["properties:10"]["betterquesting:10"]["desc:8"]
+
+		if actualQuestTitel != wantedQuestTitel or actualQuestDescription != wantedQuestDescription:
+			questbook[location][quest]["properties:10"]["betterquesting:10"]["name:8"] = wantedQuestTitel
```
This looks like it might accidentally overwrite a quest if this isn't done correctly? I can certainly imagine a quest replacing the title and description of another quest, such that when another quest attempts to find its correct title and description, it can't actually find it. Is this just impossible, or should this be mitigated?

TheScienceDemon committed:
Added localisation check

Script now checks if a quest's titel or description contains "susy.quest.db" aka checking if titel / description are already localised.

If so, but the Quest ID is wrong, the script does things like before.
(Now accept PR pls :point_right: :point_left: )

TheScienceDemon commented:
@bruberu Not sure what did first comment refers to (check [this message](https://discord.com/channels/881234100504109166/881234101103890454/1199795520223314021)) but the second thing is fixed

bruberu reviewed (approved):
I mean, you still haven't fixed the misspelling of "title", but I suppose it's fine.
